assets/RNAseq_report.Rmd

```diff
@@ -2056,6 +2067,8 @@ if (!isProvided(params$datasources)) {
 # ------------------
 # Set default params
 # ------------------
+pathway_adj_pval_threshold <- ifelse(params$pathway_adj_pval_threshold == -1, params$adj_pval_threshold, params$pathway_adj_pval_threshold)
```

Comment:
Isn't this already solved on module level? So if you don't use the pathway adjusted  p value threshold it will not be set the same as the adjusted p val threshold? (Maybe I'm missing something here)

Reply:
Oop, you're right, I overengineered :'D